ansible_collections/arista/avd/docs/installation/collection-installation.md

```diff
@@ -9,55 +9,55 @@
+- Install [arista.avd](#install-collection-from-ansible-galaxy) collection including Python requirements.
 - Modify `ansible.cfg` file to support additional [jinja2 extensions](#ansible-configuration-file)
 
 ## Install Collection from Ansible Galaxy
 
-These instructions are for regular users to install via Ansible Galaxy. To set up a development environment, please follow the [development tooling](../contribution/development-tooling.md) instructions. **arista.avd** can also be consumed using the ["Universal" container](../containers/overview.md#how-to-use-dev-containers). The **arista.avd** collection is available on the [Ansible Galaxy](https://galaxy.ansible.com/arista/avd) server and can be automatically installed on your system.
+These instructions are for regular users to install via Ansible Galaxy.
+The instructions also include installation of all Python requirements for a given version.
+To set up a development environment, please follow the [development tooling](../contribution/development-tooling.md) instructions.
+**arista.avd** can also be consumed using the ["Universal" container](../containers/overview.md#how-to-use-dev-containers).
+The **arista.avd** collection is available on the [Ansible Galaxy](https://galaxy.ansible.com/arista/avd)
+server and can be automatically installed on your system.
 
 ### Latest version
 
+These commands install all Python requirements including `ansible-core` and then installs the `arista.avd` Ansible collection including
+all required Ansible collections.
+
 ```shell
+pip install pyavd[ansible]
 ansible-galaxy collection install arista.avd
 ```
 
 !!! warning
     If you have an `ansible.cfg` file in the directory where you run `ansible-galaxy`, it may affect the directory under which the collection and dependencies will be installed.
 
+!!! warning
+    Depending of your operating system settings, `pip` might be replaced by `pip3`.
+
 ### Install a specific version
 
 ```shell
-ansible-galaxy collection install arista.avd:==4.4.0
+pip install pyavd[ansible]==4.8.0
+ansible-galaxy collection install arista.avd:==4.8.0
 ```
 
-You can specify multiple range identifiers which are split by `,`. For example, you can use the following range identifiers:
-
-- `*`: Any version, this is the default used when no range specified is set.
-- `!=`: Version isn't equal to the one specified.
-- `==`: Version must be the one specified.
-- `>=`: Version is greater than or equal to the one specified.
-- `>`: Version is greater than the one specified.
-- `<=`: Version is less than or equal to the one specified.
-- `<`: Version is less than the one specified.
-
-!!! note
-    If you are installing with a range command, you must surround the command in quotes. For example, `ansible-galaxy collection install 'arista.avd:>=4.0.0,<5.0.0'`
-
 ### Install latest `devel` version from AVD GitHub
 
 ```shell
+pip install "pyavd[ansible] @ git+https://github.com/aristanetworks/avd.git@devel#subdirectory=python-avd"
```

Comment:
Since we are also building the package I beleive this will require Python 3.10, I will test it tomorrow.
If that is the case, we should have a warning.

Reply:
Good point.

Reply:
I have added a note.